modules/ort/src/main/java/org/eclipse/sw360/antenna/ort/resolver/AbstractOrtResultResolver.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (c) Bosch Software Innovations GmbH 2019.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.eclipse.sw360.antenna.ort.resolver;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import org.eclipse.sw360.antenna.model.artifact.Artifact;
+import org.eclipse.sw360.antenna.model.artifact.facts.ArtifactCoordinates;
+import org.eclipse.sw360.antenna.model.artifact.facts.ArtifactMatchingMetadata;
+import org.eclipse.sw360.antenna.model.artifact.facts.GenericArtifactCoordinates;
+import org.eclipse.sw360.antenna.model.artifact.facts.dotnet.DotNetCoordinates;
+import org.eclipse.sw360.antenna.model.artifact.facts.java.MavenCoordinates;
+import org.eclipse.sw360.antenna.model.artifact.facts.javaScript.JavaScriptCoordinates;
+import org.eclipse.sw360.antenna.model.xml.generated.MatchState;
+
+import java.util.Optional;
+
+public abstract class AbstractOrtResultResolver implements IOrtResultResolver{
+    JsonNode result;
+
+    public AbstractOrtResultResolver(JsonNode result) {
+        this.result = result;
```

Comment:
I do not like the pattern implemented here, i.e.
```
    Artifact a = new AbstractOrtResultResolver(result).mapArtifact()
```
I would prefer the following alternative without a field variable to store the result:
```
    Artifact a = new AbstractOrtResultResolver().mapArtifact(result)
```

Reply:
Ähem, new Abstract is a no go, either, something is abstract -> not instantiable, or it is not, than Abstract is not am option for the name.